components/typography/index.css

```diff
@@ -13,6 +13,267 @@ governing permissions and limitations under the License.
+
+/* Body t-shirt sizes */
+@mixin typographyTShirtSizes .spectrum-BodyXXXL {}
+@mixin typographyTShirtSizes .spectrum-BodyXXL {}
+@mixin typographyTShirtSizes .spectrum-BodyXL {}
+@mixin typographyTShirtSizes .spectrum-BodyL {}
+@mixin typographyTShirtSizes .spectrum-BodyM {}
+@mixin typographyTShirtSizes .spectrum-BodyS {}
+@mixin typographyTShirtSizes .spectrum-BodyXS {}
+
+/* Detail / former subheadings t-shirt sizes */
+@mixin typographyTShirtSizes .spectrum-DetailXL {}
+@mixin typographyTShirtSizes .spectrum-DetailL {} 
+@mixin typographyTShirtSizes .spectrum-DetailM {} 
+@mixin typographyTShirtSizes .spectrum-DetailS {} 
+
+/* @TODO does not exists for strong or heavy for Details. Will be added in the future */
+/* 
+@mixin typographyTShirtSizes .spectrum-DetailXL--strong, detail-strong-XL { background-color: orange;}
+@mixin typographyTShirtSizes .spectrum-DetailL--strong, detail-strong-L {}
+@mixin typographyTShirtSizes .spectrum-DetailM--strong, detail-strong-M {}
+@mixin typographyTShirtSizes .spectrum-DetailS--heavy, detail-heavy-S {} */
+
+@mixin typographyTShirtSizes .spectrum-DetailXL--light, detail-light-XL {}
+@mixin typographyTShirtSizes .spectrum-DetailL--light, detail-light-L {}
+@mixin typographyTShirtSizes .spectrum-DetailM--light, detail-light-M {}
+@mixin typographyTShirtSizes .spectrum-DetailS--light, detail-light-S {}
+
+
+/* Code t-shirt sizes */
+@mixin typographyTShirtSizes .spectrum-CodeXL, code-XL, true {
+  font-family: var(--spectrum-code-xl-text-font-family);
+}
+@mixin typographyTShirtSizes .spectrum-CodeL, code-L, true {
+  font-family: var(--spectrum-code-l-text-font-family);
+}
+@mixin typographyTShirtSizes .spectrum-CodeM, code-M, true {
+  font-family: var(--spectrum-code-m-text-font-family);
+}
+@mixin typographyTShirtSizes .spectrum-CodeS, code-S, true {
+  font-family: var(--spectrum-code-s-text-font-family);
+}
+@mixin typographyTShirtSizes .spectrum-CodeXS, code-XS, true {
+  font-family: var(--spectrum-code-xs-text-font-family);
+}
+
+
+.spectrum-Typography {
+  @mixin typographyTShirtMargins .spectrum-HeadingXXXL {}
+  @mixin typographyTShirtMargins .spectrum-HeadingXXL {}
+  @mixin typographyTShirtMargins .spectrum-HeadingXL {}
+  @mixin typographyTShirtMargins .spectrum-HeadingL {}
+  @mixin typographyTShirtMargins .spectrum-HeadingM {}
+  @mixin typographyTShirtMargins .spectrum-HeadingS {}
+  @mixin typographyTShirtMargins .spectrum-HeadingXS {}
+  @mixin typographyTShirtMargins .spectrum-HeadingXXS {}
+
+  @mixin typographyTShirtMargins .spectrum-BodyXXXL {}
+  @mixin typographyTShirtMargins .spectrum-BodyXXL {}
+  @mixin typographyTShirtMargins .spectrum-BodyXL {}
+  @mixin typographyTShirtMargins .spectrum-BodyL {}
+  @mixin typographyTShirtMargins .spectrum-BodyM {}
+  @mixin typographyTShirtMargins .spectrum-BodyS {}
+  @mixin typographyTShirtMargins .spectrum-BodyXS {}
+}
+
+
+/* T-Shirt sizes for .spectrum-Article */
+.spectrum-Article {
```

Comment:
Serif is an option independent of an Article. For the update, I'm not sure it makes any sense to retain this old wrapper concept 🤔.  @GarthDB pinging for input.

Reply:
@GarthDB  @NateBaldwinDesign we have 2 options here:
1. Keep it wrapped in this `.spectrum-Article` wrapper class and duplicate all the rules outside in the root level. This will increase file size.
2. Remove this wrapper class and move all of the classes in the root level.  This will be most efficient in class name reduction. 

Let me know what you prefer.

Reply:
I'd default to @NateBaldwinDesign on this. The wrapper is a convenience if everything in the article would use the serif if that's not the case, `article` should be a variant and not a wrapper.

Reply:
Thanks @GarthDB . I will remove the wrapper class and update the documentation.